app/src/main/java/com/globant/example/mentorapp/home/domain/model/EventApiResponseEntity.java

```diff
@@ -0,0 +1,86 @@
+package com.globant.example.mentorapp.home.domain.model;
+
+import com.squareup.otto.Produce;
+
+import java.util.List;
+
+/**
+ * Class to manage otto responses from ApiClient
+ * Created by oscar.amezquita on 13/06/2017.
+ */
+
+public class EventApiResponseEntity<T> {
+
+    public static final int HTTP_OK = 200;
+    public static final int CONNECTION_ERROR = -1;
+    private final int responseCode;
+    private final String responseText;
+    private final List<T> list;
+
+    public EventApiResponseEntity(int responseCode, String responseText, List<T> list) {
```

Comment:
Make it private.
If you're following Builder pattern, this constructor should only receive a Builder instance.